ledger/ledger-on-sql/src/main/scala/com/daml/ledger/on/sql/Database.scala

```diff
@@ -200,7 +219,14 @@ object Database {
 
     def migrate(): Database = {
       flyway.migrate()
-      new Database(system.queries, readerConnectionPool, writerConnectionPool, metrics)
+      new Database(
```

Comment:
Minor: since you have a couple of parameters with the same type it could probably make sense to use named parameters.

Reply:
Good idea. Done.